lucille-examples/lucille-distributed-example/src/test/java/com/kmwllc/lucille/distributed/TestConfigs.java

```diff
@@ -0,0 +1,12 @@
+package com.kmwllc.lucille.distributed;
+
+import com.kmwllc.lucille.test.TestConfUtils;
+import org.junit.Test;
+
+public class TestConfigs {
```

Comment:
suggest changing all TestConfigs classes to ConfigValidationTest

Reply:
changed in [e5cdd5e](https://github.com/kmwtechnology/lucille/pull/196/commits/e5cdd5eb8515f9058aef636b86008f676f4b489b)